ci/jenkins/jobs/projects.yaml

```diff
@@ -0,0 +1,70 @@
+- project:
+    # ghpr_auth, antrea_admin_list, antrea_org_list and antrea_white_list
+    # should be defined as a global variable somewhere else
+    name: theia
+    git_credentials_id: ANTREA_GIT_CREDENTIAL
+    org_repo: antrea-io/theia
+    jobs:
+      - '{name}-{test_name}-for-pull-request':
+          test_name: e2e
+          node: 'antrea-test-node'
+          description: 'This is the {test_name} test for {name}.'
+          branches:
+          - ${{sha1}}
+          builders:
+            - builder-e2e
+          trigger_phrase: ^(?!Thanks for your PR).*/test-theia-(e2e|all).*
+          white_list_target_branches: []
+          allow_whitelist_orgs_as_admins: true
+          admin_list: '{theia_admin_list}'
+          org_list: '{theia_org_list}'
+          white_list: '{theia_white_list}'
```

Comment:
Better to update ci/README.md to describe Jenkins E2E test. You can refer to Antrea ci/README.md as example. We need to explain the trigger phrase, webhook, a hidden defaults.yaml, where to check test result, and how to render and update the Jobs.